table.go

```diff
@@ -210,6 +213,22 @@ func (t *Table) SetAlignment(align int) {
 	t.align = align
 }
 
+func (t *Table) SetColumnAlignment(keys []int) {
```

Comment:
doesn't seem safe to call this multiple times. is that intended?

Reply:
That's the behavior of all other `Set*` functions in this pkg.
I suspect that's because things are usually added column-by-column.